cautious.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: cautious
+spec:
+  shortDescription: Cautiously operate inside clusters
```

Comment:
I think the short description can be better. Maybe something like:

> Prevents accidental kubectl commands for contexts

Reply:
Now it is using the description you kindly provided.